content/en/docs/concepts/architecture/controller.md

```diff
@@ -89,25 +87,25 @@ in your cluster, then that controller needs something outside the
 current cluster to set up new Nodes when needed.
 
 Controllers that interact with external state find their desired state from
-the API server, then communicate directly with an external system to bring
+the API server and communicate directly with an external system to bring
 the current state closer in line.
 
-(There actually is a [controller](https://github.com/kubernetes/autoscaler/)
-that horizontally scales the nodes in your cluster.)
+There actually is a [controller](https://github.com/kubernetes/autoscaler/)
+that horizontally scales the nodes in your cluster.
 
 The important point here is that the controller makes some change to bring about
-your desired state, and then reports current state back to your cluster's API server.
+your desired state and reports the current state back to your cluster's API server.
```

Comment:
Not sure about this change. The original text described a loose sequence of events?